### PR TITLE
Users are persons

### LINKED ISSUE #419: Rename "User" to "Person"
All occurences of the term "User" in the context of identities should be replaced by "Person".

Rationale:

* user of what? Applications have users, but protocols don't (not in the same sense at least)
* There's a derogative connotation to the term: users are these mostly clueless beings who shallt be shielded from the ways of the komputermachine
* We are indeed talking about "personal identities" in the spec, which is more precise terminology: in the same sense RFC4880 describes how User IDs (there: only name + email) can be _bound_ to an identity, extensibility of the identity document payload admits _binding_ more than one external identifier (including _user_ identifiers of other systems) to the identity.

### PR DESCRIPTION
**Pull request by kim**
Fixes #419

### CODE REVIEW

File: librad/src/identities/payload.rs [hunk 1]
```diff
@@ -655,9 +655,9 @@ mod tests {
         }
     }
 
-    fn gen_user_payload() -> impl Strategy<Value = UserPayload> {
-        (any::<User>(), proptest::option::of(any::<UpstreamUser>())).prop_map(|(user, up)| {
-            let mut u = UserPayload::new(user);
+    fn gen_user_payload() -> impl Strategy<Value = PersonPayload> {
```
Comment (FintanH) [inline librad/src/identities/payload.rs:658]: Dunno how far you want to go with renaming, but the function names still mention `user`
```suggestion
    fn gen_person_payload() -> impl Strategy<Value = PersonPayload> {
```

File: librad/src/identities/payload.rs [hunk 2]
```diff
@@ -681,13 +681,13 @@ mod tests {
...
         prop_oneof![
-            gen_user_payload().prop_map(SomePayload::User),
+            gen_user_payload().prop_map(SomePayload::Person),
             gen_project_payload().prop_map(SomePayload::Project)
         ]
     }
 
-    fn gen_user_delegations() -> impl Strategy<Value = UserDelegations> {
-        proptest::collection::btree_set(gen_public_key(), 1..32).prop_map(UserDelegations)
+    fn gen_user_delegations() -> impl Strategy<Value = PersonDelegations> {
```
Comment (FintanH) [inline librad/src/identities/payload.rs:689]: Here too:
```suggestion
    fn gen_person_delegations() -> impl Strategy<Value = PersonDelegations> {
```

File: librad/src/identities/git.rs [hunk 3]
```diff
@@ -98,8 +98,8 @@ impl<'a, T: 'a> From<&'a git2::Repository> for Identities<'a, T> {
 }
 
 impl<'a, T: 'a> Identities<'a, T> {
-    /// Convenience to specialise `T` to [`User`].
-    pub fn as_user(&self) -> Identities<'_, User> {
+    /// Convenience to specialise `T` to [`Person`].
+    pub fn as_user(&self) -> Identities<'_, Person> {
```
Comment (FintanH) [inline librad/src/identities/git.rs:102]:
```suggestion
    pub fn as_person(&self) -> Identities<'_, Person> {
```

File: librad/src/identities/git.rs [hunk 4]
```diff
@@ -708,16 +708,16 @@ impl<'a> Identities<'a, Project> {
 
     fn updated_user(
```
Comment (FintanH) [inline librad/src/identities/git.rs:709]:
```suggestion
    fn updated_person(
```

**Review by FintanH — CHANGES_REQUESTED** [review]
There's mention of "users" in the identity resolution doc as well, for example in the second paragraph here https://github.com/radicle-dev/radicle-link/blob/user-is-person/docs/rfc/identity_resolution.md#motivation. It would make sense to keep that in line with the change in language here.

**Comment by kim** [comment]
> in the identity resolution doc

Yaah, but that also uses "entities" -- as that is not "official specification", I'd rather leave it as is. I've updated [identities.md](./docs/spec/identities.md)

**Comment by kim** [comment]
I also left seed stuff untouched. Not sure... that seems more like an "upstream server" at the moment.

**Comment by FintanH** [comment]
> Yaah, but that also uses "entities" -- as that is not "official specification", I'd rather leave it as is. I've updated identities.md

Fair enough :+1: I can rework it to make it more official at some point :)

**Review by FintanH — DISMISSED** [review]
I persist therefore I am

**Review by xla — APPROVED** [review]
🏮 🕷 🔧 ☑️

**Review by FintanH — APPROVED** [review]
I'm persistent therefore I am

**Pull request merged**